src/section-negative-numbers.mbx

```diff
@@ -344,7 +344,7 @@
                             <p><var name="84" width="10" /></p>
                         </li>
                         <li>
-                            <p>Find the product <m>285(0)</m>.</p>
+                            <p>Find the product <m>-285(0)</m>.</p>
```

Comment:
Are you presuming multiplication by 0 was already discussed in the first section? I don't think that it was.